### ISSUE TITLE
Inconsistency in icon/warning coloring

### ISSUE THREAD

**Issue by frenck** [zwave-js/zwave-js-ui#3242]
### Checklist

- [X]  I am not using Home Assistant. Or: a developer has told me to come here.
- [X] I have checked the troubleshooting section and my problem is not described there.
- [X] I have read the changelog and my problem is not mentioned there.

### Deploy method

Docker

### Z-Wave JS UI version

8.23.1

### ZwaveJS version

Not relevant

### Describe the bug

First of all: Sorry, this isn't really a bug but more of a suggestion. I didn't find feature request fitting either, especially considering this suggestion could put users on the wrong track (and maybe thus closer to a bug).

Anyways, I had a discussion with some people earlier today on why they would use healing a network. Many people actually don't know what it does, and it actually won't fix their problems.

At that point, I was looking at this screen: 

![image](https://github.com/zwave-js/zwave-js-ui/assets/195327/c86c1eaa-678c-420b-a110-c2b29b80fd73)

The coloring of icons in this screen, gives a signal to the end-user, in which cases the red can be perceived as destructive while the orange warns the users for potential danger; blue seems to take the neutral in this case.

Looking at the screenshot, it seems the coloring is inconsistent or maybe even falsy towards the feeling created.

For example, I've highlighted one, which is removing failed nodes. This is a destructive action, that is colored as neutral. But it could maybe re-evaluated for other things too. For example, Healing has performance degradations that may take a long time and can potentially cripple your network for hours, not sure if neutral is fitting in that case.

TL;DR: It might be worth revisiting the signalling used on this page to improve the UX.

../Frenck

### To Reproduce

See above

### Expected behavior

See above

**Comment by robertsLando** [comment]
So based on this I think the only once that may need a color change are
- Remove failed nodes. (maybe it should be orange?)
- Heal Network (orange too?)

@AlCalzone opinions? I think both of them have a confirmation message anyway that explains the potential issues of them

**Comment by AlCalzone** [comment]
Depends on what the colors are meant to indicate. IMO this scheme makes sense:
- Blue: neutral
- Yellow: potential danger or service interruption
- Red: definitely destructive

In that case:\
**Yellow**
- heal network
- re-interview all
- soft-reset
- shutdown
- NVM backup/restore (although possibly destructive)
- OTW update (although possibly destructive)

**Red**
- hard reset
- remove failed
